### ISSUE TITLE
Add REHL 8 support

### ISSUE THREAD

**Issue by slopp** [rstudio/r-builds#26]
To support the addition of REHL 8 as a supported distro in our pro products

cc: @koverholt who might be trying this out and could have instructions

**Comment by koverholt** [comment]
As a quick test, I attempted to install the RHEL 7 binaries for R on a RHEL 8 machine.

I installed the following:

```
sudo yum install \
    atlas-devel \
    bzip2-devel \
    gcc \
    gcc-gfortran \
    libSM \
    libXmu \
    libXt \
    libcurl-devel \
    libicu-devel \
    libtiff \
    openssl-devel \
    pango \
    pcre-devel \
    pcre2-devel \
    perl-Digest-MD5 \
    tcl \
    tk \
    xz-devel \
    zlib-devel \
    @'Development Tools'
```

And tried running R via `/opt/R/3.6.1/bin/R --version` and got the following error:

```
/opt/R/3.6.1/lib/R/bin/exec/R: error while loading shared libraries: libgfortran.so.3: cannot open shared object file: No such file or directory
```

I got past that error by installing `compat-libgfortran`

Then ran into the following error:

```
/opt/R/3.6.1/lib/R/bin/exec/R: error while loading shared libraries: libreadline.so.6: cannot open shared object file: No such file or directory
```

And tried installing `readline` and `readline-devel`, but RHEL 8 does not provide libreadline 6. And there is no EPEL for RHEL 8 at the moment.

**Comment by ricardofandrade** [comment]
CentOS8 is planned for a September 24 release:
https://wiki.centos.org/About/Building_8

**Comment by slopp** [comment]
@glin  is this closed by #38 ?
